README.md

```diff
@@ -76,11 +76,16 @@ Override any environment variable value from [.ddev/config.contrib.yaml](config.
 
 ### Changing the Drupal core version
 
-In `.ddev/config.local.yaml` set the Drupal core version:
+Use the `ddev core-version` command to set the core version environment variable and update the code, for example:
 
-```yaml
-web_environment:
-  - DRUPAL_CORE=^11
+```shell
+ddev core-version ^11
```

Comment:
I think many people will omit the carat here. Can we handle that seamlessly? If not, lets error if its not a valid constraint.

Reply:
`ddev core-version 11` is a valid constraint and works without error, though I wonder if it will give a result that isn't expected (`11` is shorthand for `11.0.0` rather than `^11`). I have added a test case for it (`ddev core-version short-major`).

An for an invalid version, that fails as part of the `ddev poser` command. I think that is probably fine, and I have added a test to assert the failure (`ddev core-version invalid`). However, if you think we should give an earlier and more concise error, we could do that.